scripts/ci/ci-equivalent-check.sh

```
@@ -21,15 +21,15 @@
 # Usage:
-#   bash scripts/ci-equivalent-check.sh
+#   bash scripts/ci/ci-equivalent-check.sh
 #
 # Exit code:
 #   0 = all gates green (safe to push)
 #   1 = at least one gate failed (DO NOT push without fixing)
 
 set -uo pipefail
 
-cd "$(dirname "$0")/.."
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)